packages/flutter/lib/src/widgets/animated_size.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:meta/meta.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+
+/// Animated widget that automatically transitions the child's size over a given
+/// duration whenever the given child's size changes.
+class AnimatedSize extends SingleChildRenderObjectWidget {
+  /// Creates a widget that animates the size of its child implicitly.
+  ///
+  /// The [curve] and [duration] arguments must not be null.
+  AnimatedSize({
```

Comment:
these two docs are slightly wrong. it's not the child's size that's animated, it's this widget's size.